src/site/content/en/progressive-web-apps/app-like-pwas/index.md

```diff
@@ -0,0 +1,438 @@
+
+<figure class="w-figure">
+  <img src="./image13.png" alt="iOS settings screen showing the 'Badges' toggle activated." width="310">
+  <figcaption class="w-figcaption">Badges are a subtle way for applications to inform about new content.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  You can set app icon badges with the <a href="https://web.dev/badging-api/">Badging API</a>.
+  This is especially useful when your PWA has some notion of "unread" items or when you need a means
+  to unobtrusively draw the user's attention back to the app.
+{% endDetails %}
+
+## Media playback trumps energy saver settings
+
+When podcast media is playing, the screen may turn off, but the system will not enter standby mode.
+Apps can optionally keep the screen awake, too, for example to display lyrics or captions.
+
+<figure class="w-figure">
+  <img src="./image21.png" alt="macOS Preferences in the 'Energy Saver' section." width="400">
+  <figcaption class="w-figcaption">Apps can keep the screen awake.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  The <a href="https://web.dev/wakelock/">Screen Wake Lock API</a> allows you to prevent the screen from turning off.
+  Media playback on the web automatically prevents the system from entering standby mode.
+{% endDetails %}
+
+## App discovery through an app store
+
+While the Podcasts app is part of the macOS desktop experience, on iOS it needs to be installed from the App Store.
+A quick search for "podcast", "podcasts", or "apple podcasts" immediately turns the app up in the App Store.
+
+<figure class="w-figure">
+  <img src="./image22.png" alt="iOS App Store search for 'podcasts' reveals the Podcasts app." width="300">
+  <figcaption class="w-figcaption">Users have learned to discover apps in app stores.</figcaption>
+</figure>
+
+{% Details %}
+{% DetailsSummary %}
+  How to do this on the web
+{% endDetailsSummary %}
+  While Apple does not allow PWAs on the App Store, on Android, you can submit your PWA
+  <a href="https://web.dev/using-a-pwa-in-your-android-app/">wrapped in a Trusted Web Activity</a>.
+  The <a href="https://github.com/GoogleChromeLabs/bubblewrap"><code>bubblewrap</code></a> script makes this a painless operation.
+  This script is also what internally powers <a href="https://www.pwabuilder.com/">PWABuilder</a>'s Android app export feature,
+  which you can use without touching the command line.
+{% endDetails %}
+
+## Conclusion
+
+PWAs have come a long way since their introduction in 2015.
+In the context of [Project Fugu 🐡](/fugu-status), we are working on closing the last remaining gaps.
+By following even only some of the pieces of advice in this article,
+you can reach that holy grail app-like native feeling and make your users forget
```

Comment:
The Holy Grail is a mythical thing that has never actually been found or achieved. I suggest rephrasing this because it currently implies PWAs will never actually achieve par with "native" apps.

Reply:
Good point. And while probably true (we'll never fully get there), it may not be the best way to phrase it.